getting_started_playbook: improve the doc (#253)
ansible/ansible-documentation: docs/docsite/rst/getting_started/get_started_playbook.rst

Complete the following steps to create a playbook that pings your hosts and prints a "Hello world" message:

> #. Create a file named ``playbook.yaml`` in your ``ansible_quickstart`` directory, which we created earlier, with the following content:

Reviewer: ```suggestion
#. Create a file named ``playbook.yaml`` in your ``ansible_quickstart`` directory, that you created earlier, with the following content:
```

Author: SGTM, committed, thanks, PTAL
